packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -38,10 +39,16 @@ export class ChatContext {
   private _logger: AzureLogger;
   private _emitter: EventEmitter;
   private typingIndicatorInterval: number | undefined = undefined;
+  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  private _messageQueue: MessageQueue | undefined = undefined;
 
-  constructor(maxListeners?: number) {
+  constructor(credential?: CommunicationTokenCredential, maxListeners?: number) {
```

Comment:
Should the credential go as the second parameter so not to disturb current users of this constructor?